proxy/src/utils/fetchPageData.ts

```diff
@@ -69,8 +69,15 @@ const getProcessedTokens = (tokens: any[]) => {
     const processedToken = { ...token } as any
 
     if (token.type === 'image' && token.text?.includes('|')) {
-      processedToken.text = token.text.split('|')[0]
-      processedToken.aspectRatio = token.text.split('|')[1]
+      const parts = token.href.split(' ')
+      processedToken.text = parts[0]
+      processedToken.aspectRatio = parts[1]
+    }
+
+    if (token.type === 'image' || token.type === 'image-static') {
+      const parts = token.href.split(' ')
+      processedToken.href = parts[0]
+      processedToken.title = parts[1].replaceAll("'", '')
```

Comment:
Are we not handling aspect ratio for static images?

Reply:
We are, but that is taken care of by the block above.

Reply:
See the JSON output examples above for aspect ratios being formatted on both types of images.